stack.yaml

```diff
@@ -1,10 +1,10 @@
 flags: {}
 packages:
 - '.'
-- 'yesod-markdown'
 # - location:
 #     git: git@github.com:robgssp/yesod-markdown
 #     commit: 1860815cf5fe193e9593a9adc7211929e803a7d1
 extra-deps:
 - ldap-client-0.1.0
+- yesod-markdown-0.10.0
```

Comment:
The way stack works is that it pulls in everything it can find from its "consistent" database (LTS Haskell), which is a lot of packages, but you need to manually add versions for things outside of this with extra-deps. ldap-client and yesod-markdown aren't in LTS Haskell so they're listed here.